dask_sql/context.py

```diff
@@ -829,17 +835,23 @@ def _get_ral(self, sql):
         except DFParsingException as pe:
             raise ParsingException(sql, str(pe)) from None
 
-        rel = nonOptimizedRel
-        logger.debug(f"_get_ral -> nonOptimizedRelNode: {nonOptimizedRel}")
-        # Optimization might remove some alias projects. Make sure to keep them here.
-        select_names = [field for field in rel.getRowType().getFieldList()]
+        # Optimize the `LogicalPlan` or skip if configured
+        if dask_config.get("sql.skip_optimize"):
+            rel = nonOptimizedRel
+        else:
+            try:
+                rel = self.context.optimize_relational_algebra(nonOptimizedRel)
+            except DFOptimizationException as oe:
+                rel = nonOptimizedRel
```

Comment:
Nitpicky, but would it be more direct to make this config option `optimize` and have it do the opposite of what it's doing now?

```suggestion
        if dask_config.get("sql.optimize"):
            try:
                rel = self.context.optimize_relational_algebra(nonOptimizedRel)
            except DFOptimizationException as oe:
                rel = nonOptimizedRel
        else:
            rel = nonOptimizedRel
```